exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,4 +1,7 @@
-// Declare your function first
+function divide(num1,num2){
+return num1/num2;
```

Comment:
Hi Adem, great work, my only small suggestion would be to use consistent indenting across your work (eg your "returns"), so that it reads easier.

Reply:
Thanks for your help Bisi.I will do it.